Uzira.rte/Actors/Brains/Uzira/Armour.lua

```diff
@@ -0,0 +1,19 @@
+function OnAttach(self, parent)
+	if parent then
+		if IsLeg(parent) and ToLeg(parent).Foot then
+			ToLeg(parent).Foot = CreateAttachable(string.sub(ToLeg(parent).Foot.PresetName, 1, -2) .. "B");
+		elseif IsArm(parent) then
+			self.origWoundName = ToArm(parent):GetEntryWoundPresetName();
+			ToArm(parent):SetEntryWound("Dent Metal Light", "Base.rte");
+		end
+	end
+end
+function OnDetach(self, parent)
+	if parent then
+		if IsLeg(parent) and ToLeg(parent).Foot then
+			ToLeg(parent).Foot = CreateAttachable(string.sub(ToLeg(parent).Foot.PresetName, 1, -2) .. "A");
```

Comment:
I believe this triggers an assert because it's making an attachable while one is being deleted and there's no addedattachables setup. Discussed with 4zK, needs to be so it adds a script that adds the attachable or something.